File: lista 2/python/fazendogol.py
z,g = map(str, input().split())
d, c = map(str, input().split())
if(z == d):
    print("Driblado")
    if(c == g):
        print("Gol")
    else:
        print("...e o goleiro pega")
else:
    print("Bloqueado \n")

File: lista 2/cpp/fazendogol.cpp
#include <iostream>
using namespace std;
int main(){
    char z,g,d,c;
    cin>>z>>g;
    cin>>d>>c;

    if (z==d){
        cout<<"Driblado"<<endl;
        if(c == g){
            cout<<"Gol";
        }
        else{
            cout<<"...e o goleiro pega"<<endl;
        }
        
    }
    else{
        cout<<"Bloqueado"<< endl;
        cout<<" ";
    }

}
